features/steps/push.py

```diff
@@ -0,0 +1,35 @@
+from behave import step, use_step_matcher
+import subprocess as sp
+import uuid
+import requests
+
+use_step_matcher("re")
+
+# TODO: Quay API to remove particular application?
```

Comment:
This may be a question for forum-quay, unless @kevinrizza has already asked about this.

Reply:
Hey @SamiSousa, thanks for letting me know this. This is still WIP code so I will make sure this is removed when it's finalized.